### PR TITLE
Rhmap 8429 switch to stdout

### PR DESCRIPTION
**Pull request by nialldonnellyfh**
# Motivation

Currently, gitlab-shell defaults to logging everything out to a file. This is not useful for aggregated logging as it is not visible to fluentd.

gitlab-shell is unique as there are 2 processes running in the container:

- The ssh daemon keeping the ssh connection alive  (and also the docker container)
- The execution of a gitlab-shell command over ssh (from fh-scm)

As the sshd process is executed by the Docker file using the ./bin/start.sh, the stdout from that process is what is output to the docker logs. 

Therefore, this process must be tailing the logs of the gitlab-shell processes in order to be able to see the logs in the docker STDOUT.

# Changes

- Added a fifo pipe that can be written to using the existing logging functionality. This also stops logs growing in size.
- Adding a tail command to start.sh to tail the pipe

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by maleck13** [comment]
@nialldonnellyfh this looks reasonable. We should also update the fh-openshift-template to set these vars from params

File: bin/start.sh [hunk 1]
```diff
@@ -61,6 +61,12 @@ echo "source /opt/rh/rh-ruby23/enable" > /home/git/.bashrc
 ## Map PV repositories
 ln -sf $GIT_REPO_ROOT/repositories /home/git/repositories
 
+# Creating a pipe so that gitlab-shell processes can write into something that does not increase in size
+mkfifo /home/git/gitlab-shell/gitlab-shell.log
```
Comment (maleck13) [inline bin/start.sh]: @nialldonnellyfh nice 👍

Comment (mmurphy) [inline bin/start.sh]: might be worth outputing some logging to say it's starting this.
does the output from gitlab have to be differenciated from the output of sshd? Probably not, just asking the question.
What happens if the "tail" command exits? maybe ```( tail -f /home/file.log ; echo "logging stopped unexpectedly?") &```

Comment (nialldonnellyfh) [inline bin/start.sh]: Yes, will add a message for starting and exiting a tail. Thanks.

The gitlab-shell logging output looks very different to the sshd output so it is already easy to differentiate.

**Pull request merged**